src/components/fields/ArrayField.js

```diff
@@ -366,15 +193,18 @@ class ArrayField extends Component {
       disabled,
       readonly,
       autofocus,
-      registry = getDefaultRegistry(),
+      registry,
       onBlur,
       onFocus,
       idPrefix,
-      rawErrors,
+      errors,
```

Comment:
I don't see the benefit that renaming `errors` to `rawErrors` would provide.

I understand that the naming would be more clear, but the downside is that this introduces breaking changes.

Avoiding breaking changes means that this PR is easier to merge, easier to release and more people will get benefits earlier.

Reply:
I wanted to mainly avoid adding another template (ErrorList from old SchemaField). Because component based `errors` needs UI component `ErrorList` which was in the old `SchemaField.js`. But sure we can take that back it probably means that I need to create another template for this ErrorList. What would be its name? `ErrorListTemplate` is taken by the template for list of all errors visible at the top of the form. So it should be the name which doesn't confuse devs.

Reply:
I don't really understand this argument. What are we using this `errors` and how is it different from `rawErrors` that we had before?